src/views/GovernanceReviewTeam/Actions/RequestEdits.tsx

```diff
@@ -1,52 +1,146 @@
+  );
+
   return (
-    <FormProvider<RequestEditsFields> {...form}>
-      <ActionForm
-        systemIntakeId={systemIntakeId}
-        title={t('requestEdits.title')}
-        description={t('requestEdits.description')}
-        breadcrumb={t('requestEdits.breadcrumb')}
-        successMessage={t('requestEdits.success')}
-        modal={{
-          title: t('requestEdits.modal.title'),
-          content: (
-            <Trans
-              i18nKey="action:requestEdits.modal.content"
-              // TODO: Replace formName with value from "Which form needs edits" field
-              values={{ formName: 'Draft Business Case' }}
-            />
-          )
-        }}
-        onSubmit={onSubmit}
-      >
-        {/* Action fields here */}
-      </ActionForm>
-    </FormProvider>
+    <>
+      <FormProvider<RequestEditsFields> {...form}>
+        <ActionForm
+          systemIntakeId={systemIntakeId}
+          title={t('requestEdits.title')}
+          description={t('requestEdits.description')}
+          breadcrumb={t('requestEdits.breadcrumb')}
+          successMessage={t('requestEdits.success', {
+            formName: intakeFormStepName
+          })}
+          modal={{
+            title: t('requestEdits.modal.title'),
+            content: (
+              <Trans
+                i18nKey="action:requestEdits.modal.content"
+                values={{
+                  formName: intakeFormStepName
+                }}
+              />
+            )
+          }}
+          onSubmit={submit}
+        >
+          <Controller
+            name="intakeFormStep"
+            control={control}
+            rules={{ required: t<string>('form:inputError.makeSelection') }}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup error={!!error}>
+                <Label
+                  htmlFor="intakeFormStep"
+                  hint=""
+                  error={!!error}
+                  className="text-normal"
+                >
+                  {t('requestEdits.label.intakeFormStep')}
+                  <RequiredAsterisk />
+                </Label>
+                {error && (
+                  <ErrorMessage>
+                    {t('form:inputError.makeSelection')}
+                  </ErrorMessage>
+                )}
+                <Dropdown
+                  id="intakeFormStep"
+                  data-testid="intakeFormStep"
+                  {...field}
+                  ref={null}
+                >
+                  <option>- Select -</option>
+                  {[
+                    SystemIntakeFormStep.INITIAL_REQUEST_FORM,
+                    SystemIntakeFormStep.DRAFT_BUSINESS_CASE,
+                    SystemIntakeFormStep.FINAL_BUSINESS_CASE
+                  ].map(val => (
+                    <option key={val} value={val}>
+                      {t(`requestEdits.option.intakeFormStep.${val}`)}
+                    </option>
+                  ))}
```

Comment:
Getting a bit of unwanted behavior here.  If attempted to submit without a select dropdown option, the form validation fires as expected.  The `- Select -` option is still selectable (I don't think it should be after initial selection?). This also causes the validation to not trigger if an option is selected then the `- Select -` option is reselected. 

Setting a default value on the form and having the `- Select -` option be disabled can work, but it makes typing the input and output a bit more complex

Maybe check with Zoe if the `- Select -` should be reselectable, and if so update the validation to handle this.

Reply:
Thanks for catching that! The default now has a value for validation to catch.